python/TestHarness/schedulers/Scheduler.py

```diff
@@ -99,20 +99,13 @@ def __init__(self, harness, params):
         # Private set of jobs currently running
         self.__active_jobs = set([])
 
-        # Jobs that are taking longer to finish than the alloted time are reported back early to inform
-        # the user 'stuff' is still running. Jobs entering this set will not be reported again.
+        # jobs that are taking longer to finish than the alloted time are reported back early to inform
+        # the user 'stuff' is still running. jobs entering this set will not be reported again.
```

Comment:
```suggestion
        # Jobs that are taking longer to finish than the alloted time are reported back early to inform
        # the user 'stuff' is still running. Jobs entering this set will not be reported again.
```

Victim of find and replace?